remoteworkitem/trackeritem_repository.go

```diff
@@ -129,27 +129,26 @@ func upsert(ctx context.Context, db *gorm.DB, workItem workitem.WorkItem) (*work
 		return nil, errors.WithStack(err)
 	}
 	var resultWorkItem *workitem.WorkItem
+	c := workItem.Fields[workitem.SystemCreator]
+	var creator uuid.UUID
+	if c != nil {
+		if creator, err = uuid.FromString(c.(string)); err != nil {
+			return nil, errors.Wrapf(err, "Failed to convert creator id into a UUID: %s", err.Error())
```

Comment:
ah yes, sorry about that ! I remember you already talked about it, so it's an habit (starting an error message with a capital letter) that I need to forget!